### ISSUE TITLE
Mailchimp Integration - Opt In Emails not being triggered.

### ISSUE THREAD

**Issue by mark-chief** [verbb/formie#516]
**Description**
Double Opt-in is enabled on my audience, and in the mailchimp Integration settings in Formie but the opt in emails are not sending.

Queue manager logs look to be OK??

```
2021-07-07 14:28:40 [-][1][-][info][craft\queue\QueueLogBehavior::beforeExec]  [434117] Triggering form “mailchimp” integration. (attempt: 1, pid: 18421) - Started
2021-07-07 14:28:40 [-][1][-][info][craft\queue\QueueLogBehavior::afterExec]  [434117] Triggering form “mailchimp” integration. (attempt: 1, pid: 18421) - Done (time: 0.185s)
```

Are you aware of any other logs i can check to try and debug? I cant see any logs in maichimp itself.

Many Thanks

**Additional info**
- Plugin version: 1.4.8
- Craft version: 3.6.16
- Multi-site:  No

**Additional context**

**Comment by engram-design** [comment]
Hmmm, so what Formie does is send the same payload to Mailchimp, but if "Use Double Opt in" is enabled, it'll set the status of the contact being subscribed as `pending`, rather than `subscribed`.

Having said that, from my research, it looks like we should send this as `status` and `status_if_new`. I've updated this for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.9](https://github.com/verbb/formie/releases/tag/1.4.9)
